frontend_tests/node_tests/popovers.js

```diff
@@ -98,6 +96,7 @@ function make_image_stubber() {
 
 function test_ui(label, f) {
     run_test(label, (override) => {
+        override(popovers, "clipboard_enable", noop);
```

Comment:
This does not avoid `__Rewire__`. On an ES6 module, `override` calls `__Rewire__`, as that’s the only way it can work.